bin/BuildPackages.sh

```diff
@@ -130,8 +109,12 @@ notice \
 "Note that many GAP packages require extra programs to be installed," \
 "and some are quite difficult to build. Please read the documentation for" \
 "packages which fail to build correctly, and only worry about packages" \
-"you require!" \
-"Logging into ./$LOGDIR/$LOGFILE.log"
+"you require!"
+
+echo_run() {
+  echo "$@"
```

Comment:
Is this for testing pruposes? Will it be later removed from the script? If not, then maybe we could add something to make clear that this is now a command we are running instead of an output of some command. E.g.
````echo "Running     " "$@"````